Stratis.Bitcoin.Features.BlockStore/LoopSteps/ProcessPendingStorageStep.cs

```diff
@@ -43,132 +42,149 @@ internal ProcessPendingStorageStep(BlockStoreLoop blockStoreLoop, ILoggerFactory
         /// <inheritdoc/>
         internal override async Task<StepResult> ExecuteAsync(ChainedBlock nextChainedBlock, CancellationToken cancellationToken, bool disposeMode)
         {
-            this.logger.LogTrace("({0}:'{1}/{2}',{3}:{4})", nameof(nextChainedBlock), nextChainedBlock.HashBlock, nextChainedBlock.Height, nameof(disposeMode), disposeMode);
```

Comment:
Not having logs in ExecuteAsync methods of steps seems weird. It is exactly where you want to have logs if you want to be able to track which steps were executed.

Reply:
Yes, this needs to be corrected. I moved the log output to each method in ProcessPending (ProcessWhenDisposing, ProcessWhenInInIBD and ProcessWhenNotInIBD) so that I can which one executed. It is more meaningful now. I also removed the method entry syntax. I'm only concerned with which method executed.

Reply:
See above comment about DEBUG level rather than TRACE level for this.